packager/packager.cc

```diff
@@ -972,6 +1014,10 @@ struct PackageInfoFinder {
                 arg = prependInternalPackageName(ctx, move(send.getPosArg(0)));
             }
         }
+        if (ctx.state.packageDB().getLayerCount() && send.fun == core::Names::layer() && send.numPosArgs() == 1) {
+            auto layerRank = verifyLayer(ctx, core::Names::layer(), send.getPosArg(0));
+            info->layer_ = core::packages::Layer(layerRank);
+        }
         if (send.fun == core::Names::export_for_test() && send.numPosArgs() == 1) {
```

Comment:
I think the following should be error conditions:
- specifying a layer when `--stripe-packages-layers` was not specified
- failing to specify a layer when `--stripe-packages-layers` is specified

Would want tests for both